stages/org.osbuild.systemd.unit

```diff
@@ -51,6 +62,12 @@ SCHEMA = r"""
         }
       }
     }
+  },
+  "unit-type": {
+    "type": "string"
+    "enum": ["system", "global"]
+    "default": "system",
```

Comment:
The commit message says the default is `global` but it should say it's `system`.